pyproject.toml

```diff
@@ -1,62 +1,26 @@
 [tool.poetry]
 name = "blurring_as_a_service"
-version = "1.0.2"
+version = "1.0.3"
 description = ""
 authors = [
     "Sebastian Davrieux <s.davrieux@amsterdam.nl>",
-    "Diana Epureanu <d.epureanu@amsterdam.nl>",
-    "Chris Eijgenstein <c.eijgenstein@amsterdam.nl>",
-    "Andrea Lombardo <a.lombardo@amsterdam.nl>"
+    "Andrea Lombardo <a.lombardo@amsterdam.nl>",
```

Comment:
I think we should keep the names of Chris and Diana (but perhaps without e-mail address). There is still their code (and effort) in this project right?